models/redis.js

```diff
@@ -388,6 +413,21 @@ var getProgressForKeys = function(keys){
     return dfd.promise;
 };
 
+var getDelayTimeForKeys = function(keys){
+    var dfd = q.defer();
+    var multi = [];
+    for(var i = 0, ii = keys.length; i < ii; i++){
+        multi.push(["zscore", "bull:"+keys[i].type+":delayed", keys[i].id]);
+    }
+    redis.multi(multi).exec(function(err, results){
+        for(var i = 0, ii = keys.length; i < ii; i++){
+            keys[i].delayUntil = new Date(Math.floor(results[i]/0x1000));
```

Comment:
This comes from bull internals. Took me a while to figure it out as well, so I guess an explanation would be in place here. I'll add a comment about it